java/system-test/system-test-dsl/src/main/java/sleeper/systemtest/dsl/instance/SystemTestParameters.java

```diff
@@ -343,6 +355,7 @@ public Builder loadFromSystemProperties() {
                     .forceRedeploySystemTest(getBooleanProperty("sleeper.system.test.force.redeploy", false))
                     .forceRedeployInstances(getBooleanProperty("sleeper.system.test.instances.force.redeploy", false))
                     .forceStateStoreClassname(getOptionalProperty("sleeper.system.test.force.statestore.classname").orElse(null))
+                    .createMultiPlatformBuilder(getBooleanProperty("sleeper.system.test.create.multi.platform", false))
```

Comment:
The false here means it won't create the builder if by default. I think this needs to default to true.